rclcpp/test/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ find_package(test_msgs REQUIRED)
 
 include(cmake/rclcpp_add_build_failure_test.cmake)
 
-add_subdirectory(benchmark)
+# add_subdirectory(benchmark)
```

Comment:
This change doesn't look like it was intended to be pushed?

Reply:
yeah, I used that to not have problems when checking coverage.

Reply:
```suggestion
add_subdirectory(benchmark)
```